src/Administration/Resources/app/administration/.eslintrc.js

```diff
@@ -201,9 +201,9 @@ module.exports = {
                 'vue/no-deprecated-slot-attribute': ['error'],
                 'vue/no-deprecated-slot-scope-attribute': ['error'],
                 // @deprecated v.6.7.0.0 - will be error in v.6.7
-                'sw-deprecation-rules/no-deprecated-components': ['warn', 'disableFix'],
+                'sw-deprecation-rules/no-deprecated-components': ['error'],
```

Comment:
This would cause linting errors. You need to revert it to warn and disableFix

Reply:
Thanks! I forgot to change back it